src/commands/info/releasenotes/display.ts

```diff
@@ -100,6 +104,7 @@ export default class Display extends SfdxCommand {
         this.ux.warn(`${this.id} failed: ${message}`);
 
         this.logger.trace(stack);
+        await Lifecycle.getInstance().emitTelemetry({ eventName: 'Error', Error: { message, stack } });
```

Comment:
hmm.  We do have `EXCEPTION` as a top-level event type.

Can we try setting the value for `type` like our error stuff does?

https://github.com/salesforcecli/plugin-telemetry/blob/50248fe2da68d80fb1ab613f3ece88e2ca647bf7/src/telemetry.ts#L273

Reply:
that's used here: https://github.com/salesforcecli/plugin-telemetry/blob/50248fe2da68d80fb1ab613f3ece88e2ca647bf7/src/hooks/telemetryPrerun.ts#L112

`eventName: 'COMMAND_ERROR'`